index.js

```diff
@@ -1,37 +1,42 @@
-var fs = require('fs');
+const fs = require('fs');
+const path = require('path');
 
 function WebpackTouch(options) {
-  var options = options || {}
-  this.filename = options.filename;
-  this.delay = options.delay? parseInt(options.delay) : 0;
-  this.ignoreOnError = options.ignoreOnError;
-  if (!this.filename) {
```

Comment:
We are loosing this check don't we?
we could survive without converting delay

Reply:
I brought it back. Thought `fs.readFile` would fail if filename is undefined, but it was bad idea.

Reply:
that could work, however we want it to fail at early state than the callback then it is easier to debug